From 01ad4b3f8145cc067d31f2ed5fe1f0f11567cd53 Mon Sep 17 00:00:00 2001
From: Frank Mori Hess <fmhess@users.sourceforge.net>
Date: Mon, 6 Apr 2009 15:13:04 -0400
Subject: Staging: comedi: Finish removing ni_private typedef

This fixes compilation of ni_mio_cs.c that was broken.

Signed-off-by: Frank Mori Hess <fmhess@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/ni_atmio.c  |    3 +--
 drivers/staging/comedi/drivers/ni_pcimio.c |    3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

--- a/drivers/staging/comedi/drivers/ni_atmio.c
+++ b/drivers/staging/comedi/drivers/ni_atmio.c
@@ -278,9 +278,8 @@ static const int ni_irqpin[] =
 
 struct ni_private {
 	struct pnp_dev *isapnp_dev;
-         NI_PRIVATE_COMMON
+	NI_PRIVATE_COMMON
 };
-
 #define devpriv ((struct ni_private *)dev->private)
 
 /* How we access registers */
--- a/drivers/staging/comedi/drivers/ni_pcimio.c
+++ b/drivers/staging/comedi/drivers/ni_pcimio.c
@@ -1219,9 +1219,8 @@ static struct comedi_driver driver_pcimi
 COMEDI_PCI_INITCLEANUP(driver_pcimio, ni_pci_table)
 
 struct ni_private {
-NI_PRIVATE_COMMON
+	NI_PRIVATE_COMMON
 };
-
 #define devpriv ((struct ni_private *)dev->private)
 
 /* How we access registers */
